pages/documentation/grid/formats/cim-cgmes.md

```diff
@@ -45,12 +45,21 @@ The `iidm.import.cgmes.convert-boundary` property is an optional property that d
 **iidm.import.cgmes.create-busbar-section-for-every-connectivity-node**  
 The `iidm.import.cgmes.create-busbar-section-for-every-connectivity-node` property is an optional property that defines if the CGMES importer creates an [IIDM Busbar Section](../model/index.md#busbar-section) for each CGMES connectivity node. Its default value is `false`.
 
+**iidm.import.cgmes.post-processors**
+The `iidm.import.cgmes.post-processors` property is an optional property that defines all the CGMES post-processors which are to be activated after import. By default, it is an empty list.
```

Comment:
Is there available post-processors provided by powsybl? If yes, maybe you should complete this section to give their names and their purposes?

Reply:
Yes I think there is one in powsybl-single-line-diagram but I'm not sure (@floriand-e2r do you know more about that?)

Reply:
@MioRtia You're right, there is the `CgmesDLImportPostProcessor` class to load the DL profile

Reply:
Issue #175 has been created to ask for this contribution.